ui/src/app/utils/breakLines.ts

```diff
@@ -3,14 +3,14 @@
  * that is inserted into a <pre> block.
  */
 export const breakLines = (
-  text: string,
+  text: string | null,
   breakAtSpaces = true,
   lineLength = 52
 ): string => {
   let chunks = [];
   // Check that the text includes whitespace, otherwise we'll treat it as if we
   // are not breaking at spaces.
-  if (breakAtSpaces && text.includes(" ")) {
+  if (breakAtSpaces && text?.includes(" ")) {
```

Comment:
I think it might be better to handle this failure case at the top of the fn and then treat it as a valid value for the rest of it...something like (untested):

```javascript
if (!text && text.length > 0) {
  return [];
}
```

then this line and the following one can assume that text is a string.